packages/commercetools/theme/components/Header/HeaderNav.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <SfHeaderNavigation>
+    <SfHeaderNavigationItem
+      v-for="(category, index) in categories"
+      :key="index"
```

Comment:
Don't you have some `category.id`? Using `index` from v-for as key is usually not a good idea and won't trigger rerender if category in array changes